src/main/resources/hudson/plugins/jira/pipeline/FindIssuesAtCommitLogsStep/config.jelly

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What would you want to overwrite issuePattern in runtime? Can you describe the usecase?